### ISSUE TITLE
Crash at startup: index out of range

### ISSUE THREAD

**Issue by tagesmutter** [jesseduffield/lazygit#989]
**Describe the bug**
lazygit crashes on startup on one of my repos

**To Reproduce**
Start -> crash (see screenshot for the error message)
It seems to be an array indexing issue

**Desktop (please complete the following information):**
- Arch Linux
 Lazygit v0.22.1 5611d9a3

**Additional context**
It only happens on this specific repo. I don't know what's special about it.

![lzg-error](https://user-images.githubusercontent.com/10535133/91886825-2f370b80-ec8a-11ea-82f2-49df3860367a.png)

**Comment by jesseduffield** [comment]
Interesting. Would you be able to tell me what output you get for:
```
git for-each-ref --sort=-committerdate --format="%(HEAD)|%(refname:short)|%(upstream:short)|%(upstream:track)" refs/heads
```

If it's confidential feel free to just describe what you get. Here's the first few lines of what I get running it on the lazygit repo:
```
*|master|origin/master|
 |heads/mjarkk/credentials-concept|remotes/mjarkk/credentials-concept|
 |tcell||
 |faster-move-commit-in-rebase|origin/faster-move-commit-in-rebase|
```

**Comment by tagesmutter** [comment]
I think, I have found the issue. And sorry, it's not really lazy gits fault. I'm using Dropbox and there were some Dropbox conflict files in the .git directory. This must have screwed up your counting.
I noticed this after the 'git gc' command was also failing in this repository with an error message:
error: bad ref for .git/logs/HEAD (hp's conflicted copy 2020-08-14)
There was also a conflict file in .git/logs/refs/heads/master (hps's conflicted copy 2020-08-14)

After deleting these conflict files. Lazygit works as expected!
